.github/workflows/pull_requests.yml

```diff
@@ -82,7 +82,6 @@ jobs:
           curl -Lo /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v${{ env.KUTTL_VERSION }}/kubectl-kuttl_${{ env.KUTTL_VERSION }}_linux_x86_64
           chmod +x /usr/local/bin/kubectl-kuttl
       - name: Run Helm e2e tests
-        if: false  # skipping this test for now, there's something wrong with the configuration
```

Comment:
Why are you making this change here?

Reply:
Checking what is going to happen

Reply:
And it seems to be passing now :thinking: :confused: